Sources/MenuViewController.swift

```diff
@@ -18,12 +18,45 @@
 
 import UIKit
 
-open class MenuViewController: UIViewController {
+@objc public protocol MenuViewController
+{
+    // UIViewController
+    var transitioningDelegate: UIViewControllerTransitioningDelegate? { get set }
+    var view: UIView! { get }
+    
+    // MenuViewController
+    weak var menuContainerViewController: MenuContainerViewController? { get set }
+    var navigationMenuTransitionDelegate: MenuTransitioningDelegate? { get set }
+    @objc func handleTap(recognizer: UIGestureRecognizer)
+}
 
-    public weak var menuContainerViewController: MenuContainerViewController?
-    var navigationMenuTransitionDelegate: MenuTransitioningDelegate?
+private var key_menuContainerViewController = "menuContainerViewController"
+private var key_navigationMenuTransitionDelegate = "navigationMenuTransitionDelegate"
 
-    func handleTap(recognizer: UIGestureRecognizer){
+extension UIViewController: MenuViewController
+{
+    public var menuContainerViewController: MenuContainerViewController?
```

Comment:
@truemetal Also, it exposes `menuContainerViewController` and `navigationMenuTransitionDelegate` for all UIViewControllers in the client when you use `InteractiveSideMenu`. That wouldn't be convenient from a usage perspective.